google-cloud-spanner/acceptance/spanner/client/large_data_test.rb

```diff
@@ -85,7 +85,7 @@ def random_row
   it "writes and queries bytes" do
     my_row = random_row
     db.upsert table_name, my_row
-    results = db.execute "SELECT id, string, byte, strings, bytes FROM #{table_name} WHERE id = @id", params: { id: my_row[:id] }
+    results = db.execute_sql "SELECT id, string, byte, strings, bytes FROM #{table_name} WHERE id = @id", params: { id: my_row[:id] }
```

Comment:
also its confusing whats the difference between execute_sql and execute_query?

Reply:
There is no difference, it is the same method with an aliased name. I changed some calls to `execute_sql` so that the alias was used in acceptance tests.